easybuild/easyconfigs/n/netCDF/netCDF-4.8.0-gompi-2021a.eb

```diff
@@ -0,0 +1,37 @@
+name = 'netCDF'
+version = '4.8.0'
+
+homepage = 'https://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented
+ scientific data."""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/Unidata/netcdf-c/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['aff58f02b1c3e91dc68f989746f652fe51ff39e6270764e484920cb8db5ad092']
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('CMake', '3.20.1'),
+    ('Doxygen', '1.9.1'),
+]
+
+dependencies = [
+    ('HDF5', '1.10.7'),
+    ('cURL', '7.76.0'),
+    ('Szip', '2.1.1'),
+]
+
+# HDF5 version detection is missed in netCDF 4.8.0 when HDF5_C_LIBRARY, HDF5_INCLUDE_DIR, and HDF5_HL_LIBRARY are set
+local_hdf5_version_fix = '-DHDF5_VERSION=$EBVERSIONHDF5'
```

Comment:
Shall we update the easyblock to set this parameter?

Reply:
I do not think we need to do that. There is a bug in netCDF 4.8.0 that causes the detection of the HDF5  version to be skipped (https://github.com/Unidata/netcdf-c/issues/1962). The fix for this will be in netCDF 4.8.1 and so this workaround should not be needed when we build future versions.

Reply:
I think we only need to do it here to circumvent a bug, right @branfosj?

Reply:
agreed :+1: